packages/lit-html/src/directives/cache.ts

```diff
@@ -21,13 +22,25 @@ import {
   clearPart,
   getCommittedValue,
   insertPart,
+  isCompiledTemplateResult,
   isTemplateResult,
   setCommittedValue,
 } from '../directive-helpers.js';
 
+/**
+ * @returns The template strings array for the provided template.
+ */
+function getStringsFromTemplate(
```

Comment:
Use an arrow function

Reply:
@justinfagnani For my own curiosity's sake, is the main reason for preferring arrow function here for the brevity of the implicit return?

Reply:
A few reasons:
* Consistency
* As an indicator that this function doesn't need a bindable `this`, or rather `function` should be an indicator that a function _does_ need a bindable `this`. Similar to preferring `const` so that `let` is an indicator that a variable will be reassigned.
* It's a few less characters, and because of our consistent usage it _may_ compress better in gzip (brotli likely as a pre-baked dictionary entry for `function`, but I'd have to look that up).